vimcolorschemes/vimcolorschemes
# feat: Add dark/light filter options

## Type of PR


## Description

Adds a filter option for dark/light repositories.

What was done:

- UI
- Create /dark and /light pages, as well as /light/dark with all sort routes
- Update the search query to filter by dark and light

## Related issue

Closes #49 

## QA Instructions

- Start app locally
- Click on the dark/light inputs
- See that only the repositories that have a color scheme matching the filter appear
- See that the repositories that have both background display a colorscheme with a background matching the filter first

## Added tests?


## Added documentation?
